src/containers/ProjectContentsContainer.jsx

```diff
@@ -70,7 +75,7 @@ class ProjectContentsContainer extends Component {
         >
           <ModalBody>
             <p>{this.state.translationText}</p>
-            <Dropdown split title="Language">
+            <Dropdown split title={this.state.translationLanguage || 'Language'}>
```

Comment:
This feels a bit of a hack. The Pivotal dropdown docs are literally changing as I write https://github.com/pivotal-cf/pivotal-ui/blob/master/styleguide_new/docs/components/Dropdowns.md, so I'll give it another go later.